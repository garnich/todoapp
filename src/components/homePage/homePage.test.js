import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import HomePage from './homePage'

Enzyme.configure({ adapter: new Adapter() })

describe('Test HomePage component', () => {
  it('<HomePage>', () => {
    const wrapper = shallow(<HomePage />)

    expect(wrapper).toMatchSnapshot()
  })
})
