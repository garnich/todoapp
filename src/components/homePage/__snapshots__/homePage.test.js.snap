// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test HomePage component <HomePage> 1`] = `ShallowWrapper {}`;
